test/spec/features/stack/upgrade_spec.rb

```diff
@@ -3,14 +3,16 @@
 describe 'stack upgrade' do
   context 'from file' do
     before(:each) do
+      run 'kontena stack rm --force redis'
+      run 'kontena stack rm --force links-external-linked'
       with_fixture_dir("stack/upgrade") do
-        k = run 'kontena stack install version1.yml'
-        expect(k.code).to eq(0)
+        run 'kontena stack install version1.yml'
```

Comment:
No more `k.code` exit code check? TBH these really really need some kind of `run!` that raises on non-zero exit codes, the explicit checks are terrible...

Reply:
It was a pain to figure out why the thing was saying:

```
expected 0 got 1
```

when my spec clearly said: `expect(k.code).to eq 1`, but it was the before block expectation that failed